tests/integration/optimizers/test_optimizer.py

```diff
@@ -58,14 +58,86 @@ def test_optimizer(tmpdir, config):
     validate_result(result, tmpdir)
 
 
+def test_optimizer_multi_flow(tmpdir, config):
+    multi_flow_runner = MultiFlowRunner(
+        SingleFlowRunner(
+            flow_yaml='tests/integration/optimizers/flow1.yml',
+            documents=document_generator(10),
+            request_size=1,
+            execution_method='search',
+        ),
+        SingleFlowRunner(
+            flow_yaml='tests/integration/optimizers/flow2.yml',
+            documents=document_generator(10),
+            request_size=1,
+            execution_method='search',
```

Comment:
I would put the second flow first and remove the evaluator from the second one. This demonstrates, that we only can leverage one evaluator. 

And we could also achieve this by just setting `execution_method='index'` for the first flow. Then we don't even need two flow files.

Reply:
nice idea! I changed it accordingly